From 31b19169cc13b6af0e972db38d1d43967a49ecf0 Mon Sep 17 00:00:00 2001
From: Nirbheek Chauhan <nirbheek@centricular.com>
Date: Mon, 10 Oct 2016 23:27:50 +0530
Subject: [PATCH 13/16] tests/gnome: Add missing enums.h dep to enums2.c

Was causing intermittent test failures

(cherry picked from commit 6a002a7bea8860590fd1ad54d2e12b91c5137e1f)
---
 test cases/frameworks/7 gnome/mkenums/meson.build | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/test cases/frameworks/7 gnome/mkenums/meson.build b/test cases/frameworks/7 gnome/mkenums/meson.build
index efd6b04..e01e9eb 100644
--- a/test cases/frameworks/7 gnome/mkenums/meson.build	
+++ b/test cases/frameworks/7 gnome/mkenums/meson.build	
@@ -32,7 +32,7 @@ enums_h2 = gnome.mkenums('abc2',
 
 enums_c2 = gnome.mkenums('abc2',
   sources : 'meson-sample.h',
-  depends : enums_h2,
+  depends : [enums_h1, enums_h2],
   c_template : 'enums2.c.in',
   ftail : '/* trailing source file info */',
   install_header : true,
-- 
2.10.1

